pulp_ansible/tests/functional/cli/test_collection_download_count.py

```diff
@@ -0,0 +1,125 @@
+"""Tests that Collections hosted by Pulp can be installed by ansible-galaxy."""
+from os import path
+import subprocess
+import pytest
+from urllib.parse import urljoin
+
+from pulpcore.client.pulp_ansible import AnsibleRepositorySyncURL
+from pulp_smash import api, config
+from pulp_smash.pulp3.bindings import monitor_task
```

Comment:
Please try not to use pulp_smash anymore.

`monitor_task` is available a s a pytest fixture these days.
And instead of the pulp_client fixture, a simple call to requests may do the trick, no?